.github/actions/test-khiops-install/action.yml

```diff
@@ -19,13 +21,24 @@ runs:
         OMPI_MCA_rmaps_base_oversubscribe: true
         KHIOPS_MPI_VERBOSE: true
       run: |
+        # Compute or get the khiops_env directory
+        if command -v khiops_env &>/dev/null
+        then
+          KHIOPS_ENV_DIR=$(dirname $(type -P khiops_env))
+        else
+          KHIOPS_ENV_DIR=$CONDA_ENV_BIN_DIR
```

Comment:
Where is `CONDA_ENV_BIN_DIR` defined ??

Reply:
Ah, now I see, it is in the environment ... I'd put it as an action parameter. That way all is self-contained.

Reply:
Or at least test that is defined and raise an error if not.

Reply:
If `CONDA_ENV_BIN_DIR` is not defined, then `KHIOPS_ENV_DIR` is empty, whence all `/khiops_env`, `/khiops` and `/khiops_coclustering` commands subsequently called would fail anyway, hence the CI would be red.

Do we need to complexify the CI to handle the undefined-variable error just a bit earlier?

Reply:
Done.